src/pages/governance.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { Page } from '../types';
+import Layout from '../components/Layout';
+import Article from '../components/Article';
+import Footer from '../components/Footer';
+import '../styles/article-reader.scss';
+
+export default function GovernancePage({ data }: Page): JSX.Element {
+  const { title, description } = data.page.frontmatter;
+  const { html, tableOfContents } = data.page;
+  const { authors } = data.page.fields;
+
+  return (
+    <>
+      <Layout title={title} description={description} showFooter={false}>
+        <main className="blog-container">
+          <Article
+            title={title}
+            html={html}
+            tableOfContents={tableOfContents}
+            authors={authors}
+            editPath="content/about/privacy.md"
```

Comment:
Should this be governance?

```suggestion
            editPath="content/about/governance.md"
```

Reply:
yep copy pasta mistake